### ISSUE TITLE
Shows sites as "down", but sites opening and eorking correctly

### ISSUE THREAD

**Issue by Iliya5digital** [upptime/upptime#735]
**Describe the bug**
Part of sites shows as "down" (list see  at "Additional context"). I just open these sites on Chrome browser, sites opening and works correctly. I used WIFI connection.

**Smartphone (please complete the following information):**
 - Device: Redmi 4A
 - OS: Android 7.1.2
 - Manufacturer: Xiaomi
 - Manufacturer's OS: MIUI 10.2.3.0 Stable
 - Browser: Chrome 
 - Version: 110.0.5481.65

Additional context
List of "down" sites shown:
https://wallpaperaccess.com/
https://zaberem.by/
https://www.reg.ru/
https://statonline.ru/
https://www.amd.by/
https://rutracker.org/

**Comment by shruuub** [comment]
Yeah, most of these sites are based in Belarus or Russia, and I suppose these are blocked. Otherwise, at least for rutracker, it might be some kind of bot detection system.

**Comment by AnandChowdhary** [comment]
Aah damn, I'm sorry :( I don't think there's anything we can do if GitHub IP addresses are unable to access your site. Let's keep this issue open to inform others! :)
